examples/offline_profile.py

```diff
@@ -261,12 +269,16 @@ def abort_requests():
                         default=BATCH_SIZE_DEFAULT,
                         help=f"Number of requests to run as a single batch, "
                         f"default={BATCH_SIZE_DEFAULT}")
-    parser.add_argument(
-        "--output-len",
-        type=int,
-        default=OUTPUT_LEN_DEFAULT,
-        help="Number of llm steps to run (includes prefill and decode) "
-        "- default={OUTPUT_LEN_DEFAULT}")
+
+    parser.add_argument("--max-output-len",
+                        type=int,
+                        default=OUTPUT_LEN_DEFAULT,
+                        help="Maximum output length of the requests")
+
+    parser.add_argument("--min-output-len",
+                        type=int,
+                        default=OUTPUT_LEN_DEFAULT,
+                        help="Minimum output length of the requests")
```

Comment:
could we maybe keep a `--output-len` option? thats mutually exclusive  with `--max-output-len` and `--min-output-len`?, seems a bit cumbersome if I want say all to have output-len of 8 to do `--max-output-len 8 --min-output-len 8`

Reply:
Hey. I have updated to CLI args to pass in `num_steps` directly. I believe it is better in capturing the intent. PTAL.